libs/k8s-client/src/ClientFacadeFactory/InClusterClientFacadeFactory.php

```diff
@@ -23,13 +23,13 @@ public function __construct(
         $this->credentialsPath = $credentialsPath;
     }
 
-    public function createClusterClient(): KubernetesApiClientFacade
+    public function createClusterClient(?string $namespace = null): KubernetesApiClientFacade
```

Comment:
uvažil bych jestli to nechceme udělat povinný (explicitní), ušetříme si jednu větěv

Reply:
> ušetříme si jednu větěv

Myslis jako tady v kodu (if/fallback), ze? Ja bych mozna radeji pockal, jak se to bude pouzivat. Koukal jsem ted na nejake oficialni K8S API clienty a taky podobou formu fallbacku maji. Ale je to jasneji oddeleny treba od credentials (tady mam reseni smichany credentials a default namespace v jednom miste).